circuit_maintenance_parser/parsers/zayo.py

```diff
@@ -50,9 +73,13 @@ def parse_bs(self, btags: ResultSet, data: dict):
                             break
                 elif line.text.lower().strip().startswith("reason for maintenance:"):
                     data["summary"] = self.clean_line(line.next_sibling)
+                # TODO: not all Zayo notifications include "date notice sent".
+                # Do we need to pull the date from the email headers, or what should we do here?
                 elif line.text.lower().strip().startswith("date notice sent:"):
                     stamp = parser.parse(self.clean_line(line.next_sibling))
                     data["stamp"] = self.dt2ts(stamp)
+                # TODO: not all Zayo notifications include "customer".
+                # Do we need to pull the customer name from the email subject line, or what should we do here?
```

Comment:
if the info of the customer is available in the subject, then we could use it adding a new parser to the processor